x/crosschain/keeper/cctx.go

```diff
@@ -48,15 +59,39 @@ func (k Keeper) SetCctxAndNonceToCctxAndInboundHashToCctx(
 		in.CctxIndex = append(in.CctxIndex, cctx.Index)
 	}
 	k.SetInboundHashToCctx(ctx, in)
+}
 
+func (k Keeper) UpdateZetaAccounting(
```

Comment:
Should we export these functions to other packages? If not, let's un-export most of them to retain only the essential API methods for external use.

Reply:
40fade6e75ce72da71aa17ff87625d4dd40d6890